data/base-recipes.yaml

```diff
@@ -195,15 +195,15 @@ crafting_recipes:
   work_order: false
   chapter: 6
   part:
-  - long.cord
+  - long cord
```

Comment:
❓ Have you tested one of each part that was changed (one 'long cord', one 'short pole', one 'long pole', etc.)?